client/src/components/Atoms/ListAndDetail.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { ReactNode } from 'react'
+import styled from 'styled-components'
+import { Button, Card, Classes, Colors, InputGroup } from '@blueprintjs/core'
+
+/**
+ * A set of components to render a list-and-detail panel
+ *
+ * Example usage:
+ *
+ *  <ListAndDetail>
+ *    <List>
+ *      <ListItem>Item 1</ListItem>
+ *      <ListItem selected>Item 2</ListItem>
+ *    </List>
+ *    <Detail>
+ *      {detailsForSelectedListItem}
+ *    </Detail>
+ *  </ListAndDetail>
+ */
+
+export const ListAndDetail = styled(Card)`
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  overflow-y: auto;
```

Comment:
I think the list and the detail panels should each scroll separately.

Reply:
https://github.com/votingworks/arlo/pull/1661/commits/0f91c7c368267c8d1dfe0760d7a6f5f310acaac0